src/core/optimization_container.jl

```diff
@@ -1271,8 +1291,14 @@ function _calculate_dual_variables_continous_model!(
 end
 
 function _process_duals(container::OptimizationContainer, lp_optimizer)
```

Comment:
@odow this is the implementation we followed to retrieve the "duals" of the MILP UC models. In case you have time to check it out please.